### ISSUE TITLE
[UI-FIX] Enhance "readability/understandablity" of Tokens attached to active CSC account

### ISSUE THREAD

**Issue by zenkertA** [casinocoin/tokenwallet-mobile#187]
**Section or interface element**
Wallet/overview listing active CSC accounts and Tokens.

**Reason**
Enhance "readability/understandablity" of Tokens attached/connected to a active CSC account with indent of the Tokens. Just a quick glanze will show/explain the connection between CSC accounts and Tokens.

**Description**
See edited screenshot below.

**Screenshots**
![bild](https://user-images.githubusercontent.com/39646311/74538509-aef2c800-4f3c-11ea-8e47-99075e7b11bf.png)

**Comment by t3nshi** [comment]
No, I think this is confusing. Form my point of view we should not group them like this,  because the token accounts are not tyied to the originating CSC account per se, I mean any operation you do on the originating account does not affect the token account, the token accounts are only supported by their reserve and from there the fees will be deduced. But they don't have any hierarchy or something like that. I know this is a very technical matter, but I don't see how giving this kind of view will help user operation. Of course I might be wrong.. So what other people think about it? please let us know. for instance... I think this is a way to take more space from the available space for presenting the view, I don't see it necessary.

**Comment by zenkertA** [comment]
In my opinion. Not at all confusing. LOL.

Den fre 14 feb. 2020 19:45TeNsHi SaMa <notifications@github.com> skrev:
